Intersect.Server/Entities/Resource.cs

```diff
@@ -68,7 +68,7 @@ public override void Die(bool dropItems = true, Entity killer = null)
                 if (Base.AnimationId != Guid.Empty)
                 {
                     PacketSender.SendAnimationToProximity(
-                        Base.AnimationId, -1, Guid.Empty, MapId, (byte)X, (byte)Y, (int)Directions.Up, MapInstanceId
+                        Base.AnimationId, -1, Guid.Empty, MapId, (byte)X, (byte)Y, (int)Direction.Up, MapInstanceId
```

Comment:
SendAnimationToProximity should accept direction and ints, no byte crap -- it can cast to byte in the method

Reply:
oh, yeah i missed that one and few others 😷 just did some minor but required changes for this:

![imagen](https://user-images.githubusercontent.com/17498701/223031527-138e5c63-3852-4bf5-8d88-2986e346e55a.png)


![imagen](https://user-images.githubusercontent.com/17498701/223031333-2ded0bc3-aff2-48fa-a439-9aca63ee6133.png)

![imagen](https://user-images.githubusercontent.com/17498701/223031427-3b776e4f-e571-47b8-97fb-ae05ca3601f0.png)